week1_gp6_zad3.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+using namespace std;
+
+enum class Course{
+    first =1,
+    second =2,
+    tirth = 3,
+    forth=4
+};
+
+struct Student{
+    double grade; // 8B
+    unsigned int fn; // 4B
+    char name[17]; // 2B*17=34B
+    Course course;
+};
+
+void PrintStudent(const Student& toPrint){
+    cout<<"name: "<<toPrint.name<<endl;
+    cout<<"fn: "<<toPrint.fn<<endl;
+    switch (toPrint.course){
```

Comment:
Wouldn't it look better if you extract this conversion to a separate function instead ?